pil-analyzer/src/evaluator.rs

```diff
@@ -1400,18 +1386,21 @@ fn evaluate_builtin_function<'a, T: FieldElement>(
             let Value::Integer(channel) = channel.as_ref() else {
                 panic!()
             };
-            symbols.get_input_from_channel(
+            symbols.input_from_channel(
                 u32::try_from(channel).unwrap(),
                 usize::try_from(index).unwrap(),
             )?
         }
-        BuiltinFunction::OutputByte => {
+        BuiltinFunction::OutputToChannel => {
             let byte = arguments.pop().unwrap();
```

Comment:
If we abstract it, we should do it properly:
byte -> value
fd -> channel

Reply:
oh right, ill see if i missed this in other places